src/components/SistemaLogoColour.tsx

```diff
@@ -0,0 +1,7 @@
+import { Image } from '@chakra-ui/react';
+
+export function SistemaLogoColour() {
+  return (
+    <Image src="/images/sistema_logo_colour.png" alt="Sistema Logo Colour" />
```

Comment:
This image is not well high quality / well cut out lmaooooo. I think the one I have attached should work better.
![sistema_logo_colour](https://github.com/user-attachments/assets/bf4ba11e-9d50-4194-a9d2-7775cc5d890e)

Reply:
Thank you for attaching this!